main/integration-tests/__test__/api-tests/costs/get-costs.test.js

```diff
@@ -0,0 +1,72 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('Get costs scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('Getting costs on dashboard', () => {
+    it('should fail if user is inactive', async () => {
+      const researcherSession = await setup.createResearcherSession();
+      await adminSession.resources.users.deactivateUser(researcherSession.user);
+
+      await expect(researcherSession.resources.costs.getIndexCosts()).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should pass if user is active', async () => {
+      const researcherSession = await setup.createResearcherSession();
+
+      const response = await researcherSession.resources.costs.getIndexCosts();
+
+      expect(response[0]).toHaveProperty('cost');
+      expect(response[0]).toHaveProperty('startDate');
+    });
```

Comment:
Does this require some setup to pass? Is the assumption that our pipeline account has done this manual setup?

Reply:
Yes, need to enable cost explorer and activate the 3 tags. (I've enabled the cost explorer in the e2e env today.)